drv/ArrayFIFOQueue.drv

```diff
@@ -154,13 +171,21 @@ public class ARRAY_FIFO_QUEUE KEY_GENERIC implements PRIORITY_QUEUE KEY_GENERIC,
 		return array[start];
 	}
 
-
 	@Override
 	public KEY_GENERIC_TYPE LAST() {
 		if (start == end) throw new NoSuchElementException();
 		return array[(end == 0 ? length : end) - 1];
 	}
-
+	
```

Comment:
Whitespace example

Reply:
Ok, this one i am not sure what you mean because this is the only example that clashes with your other ones.
What you mean is tabs between lines so that the curser does go back if to the beginning of the line if no code is written.
Yeah i can fix these pretty easily, but that is something I would have no guessed since you said "whitespace"

Unless you mean new lines and not whitespaces/tabs.

Reply:
Whitespace refers to tabs, spaces, newlines, etc.